astropy/io/votable/tree.py

```diff
@@ -1547,24 +1547,25 @@ def from_table_column(cls, votable, column):
         `astropy.table.Column` instance.
         """
         kwargs = {}
+        meta = column.info.meta or {}
```

Comment:
I think here I'd prefer an if statement, so the loop does not get executed unless meta actually contains something, i.e.,
```
meta = column.info.meta
if meta:
    for key ...
```